src/main/kotlin/uk/co/craigbass/pratura/math/CurrencyExtensions.kt

```diff
@@ -4,3 +4,5 @@ import java.math.BigDecimal
 
 fun String.toDecimal(): BigDecimal = BigDecimal(this)
 fun Int.toDecimal(): BigDecimal = BigDecimal(this)
+fun BigDecimal?.toCurrencyWithSymbol() = if (this == null) "£0.00" else "£${toCurrencyString()}"
```

Comment:
You can typealias in Kotlin then attach the extension method to the typealias only too...